android/build.gradle

```diff
@@ -90,7 +89,8 @@ dependencies {
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"
-  implementation 'com.github.dotintent:MultiPlatformBleAdapter:a136c3f4ac'
+  implementation 'io.reactivex.rxjava2:rxjava:2.2.17'
+  implementation "com.polidea.rxandroidble2:rxandroidble:1.17.2"
```

Comment:
still polidea?

Reply:
It's the other library. 
We'll try to refactor out of it in future.